fe/fe-core/src/main/java/com/starrocks/sql/plan/ScanNodePredicates.java

```diff
@@ -0,0 +1,71 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.sql.plan;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import com.starrocks.analysis.Expr;
+import com.starrocks.analysis.TupleDescriptor;
+import com.starrocks.catalog.PartitionKey;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+public class ScanNodePredicates {
```

Comment:
I suggest we should use specific names like 'HdfsScanNodePredicates` or `HiveScanNodePredicates`. If you use `ScanNodePredicates`, it means this predicates could also be used by OlapScanNode, but is it?

Or `LakeScanNodePredicates` works if you don't have better name. Anyway, I think `ScanNodePredicates` is a very broad term.

Reply:
Or `RemoteScanNodePredicate`?

Reply:
done, rename to HDFSScanNodePredicates